nilearn/plotting/html_surface.py

```diff
@@ -0,0 +1,584 @@
+        return mesh
+    if mesh != 'fsaverage5':
+        raise ValueError(
+            'mesh should be fsaverage5 until fsaverage fetcher pr is merged')
+    mesh = datasets.fetch_surf_fsaverage5()
+    return mesh
+
+
+def full_brain_info(stat_map, mesh='fsaverage5', threshold=None,
+                    cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
+                    vol_to_surf_kwargs={}):
+    info = {}
+    mesh = _check_mesh(mesh)
+    surface_maps = {
+        h: surface.vol_to_surf(stat_map, mesh['pial_{}'.format(h)],
+                               **vol_to_surf_kwargs)
+        for h in ['left', 'right']
+    }
+    colors, cmin, cmax, cmap, norm, abs_threshold = colorscale(
+        cmap, np.asarray(list(surface_maps.values())).ravel(), threshold,
+        symmetric_cmap=symmetric_cmap)
+
+    for hemi, surf_map in surface_maps.items():
+        sulc_depth_map = surface.load_surf_data(mesh['sulc_{}'.format(hemi)])
+        info['pial_{}'.format(hemi)] = to_plotly(mesh['pial_{}'.format(hemi)])
+        info['inflated_{}'.format(hemi)] = to_plotly(
+            mesh['infl_{}'.format(hemi)])
+
+        info['vertexcolor_{}'.format(hemi)] = _get_vertexcolor(
+            surf_map, cmap, norm, abs_threshold, sulc_depth_map)
+    cmin, cmax = float(cmin), float(cmax)
+    info["cmin"], info["cmax"] = cmin, cmax
+    info['black_bg'] = black_bg
+    info['full_brain_mesh'] = True
+    return info, colors
+
+
+def _fill_html_template(info, colors, embed_js=True):
+    as_json = json.dumps(info)
+    as_html = HTML_TEMPLATE.replace('INSERT_STAT_MAP_JSON_HERE', as_json)
+    as_html = as_html.replace('INSERT_COLORSCALE_HERE', colors)
+    as_html = add_js_lib(as_html, embed_js=embed_js)
+    return HTMLDocument(as_html)
+
+
+def view_img_on_surf(stat_map, mesh='fsaverage5',
+                     threshold=None, cmap=cm.cold_hot,
+                     black_bg=False, embed_js=True):
+    """
+    Insert a surface plot of a statistical map into an HTML page.
+
+    Parameters
+    ----------
+    stat_map : Niimg-like object, 3d
+        See http://nilearn.github.io/manipulating_images/input_output.html
+
+    mesh : str or dict, optional.
+        if 'fsaverage5', use fsaverage5 mesh from nilearn.datasets
+        if a dictionary, keys should be 'infl_left', 'pial_left', 'sulc_left',
+        'infl_right', 'pial_right', 'sulc_right', containing inflated and
+        pial meshes, and sulcal depth values for left and right hemispheres.
+
+    threshold : int, optional
+        int in [0, 100]: percentage of values to be thresholded.
+
+    cmap : str or matplotlib colormap, optional
+
+    black_bg : bool, optional (default=False)
+
+    embed_js : bool, optional (default=True)
+        if True, jquery and plotly are embedded in resulting page.
+        otherwise, they are loaded via CDNs.
+
+    Returns
+    -------
+    HTMLDocument : html page containing a plot of the stat map.
```

Comment:
We should probably not call it this way: it will confuse end users, for instance who use it in the notebook. We should also say that it is represented in the notebook.